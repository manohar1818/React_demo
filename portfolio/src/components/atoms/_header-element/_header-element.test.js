import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import HeaderElement from './_header-element';

describe("HeaderElement component",() =>{
    test("Matches the snapshot",() => {
        const headerElement = create(<HeaderElement />);
        expect(headerElement.toJSON()).toMatchSnapshot();

    })
    test("header element is rendered",() => {
        const { getByTestId, getByText } = render(<HeaderElement />);
        expect(getByTestId("headerElement")).toBeInTheDocument();
        
    });
   

});