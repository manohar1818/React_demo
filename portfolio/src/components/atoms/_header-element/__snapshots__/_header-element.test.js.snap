// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderElement component Matches the snapshot 1`] = `
<h4
  className="MuiTypography-root makeStyles-styleHeaderElement-1 MuiTypography-h4"
  data-testid="headerElement"
/>
`;
